os/arch/xtensa/src/esp32/esp32_allocateheap.c

```diff
@@ -98,17 +98,32 @@ void up_allocate_heap(FAR void **heap_start, size_t *heap_size)
 	*heap_size = (size_t)((uintptr_t)&_eheap - (uintptr_t)&_sheap);
 }
```

Comment:
this version is for development, and later we need to merge to latest public release version before we can submit PR for it. so we can ignore file mode here, and it will be handled properly when PR to public